### PR TITLE
Icon Design Update

### PR DESCRIPTION
**Pull request by ytk141**
# Boostnote Icon Design Update

boostnote icon design update

## Sidebar&NoteList
sidebar notelist

## NoteDetail__info-right
notedetail__info-right

### CODE REVIEW

**Comment by kazup01** [comment]
Thank you for contribution @ytk141 ! I will check it :)

File: boostnote.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (kazup01) [inline boostnote.json]: This file (boostnote.json) is unnecessary. Could you delete it?

**Comment by ytk141** [comment]
@kazup01 Updated!

**Comment by kazup01** [comment]
@ytk141  Thanks!

**Comment by kazup01** [comment]
Merged. I will fix CI error soon.
Thank you again @ytk141 🎉

**Pull request merged**